python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
+                     'exception type will never be reached. (%s)',
+            'E0702': '\'raise\' is used to raise Exceptions. Anything else will lead to an error. '
+                     '(Raising %s)',
+            'E0704': '\'raise\' may only be called without an expression inside an \'except\' block.',
+            'E0710': '\'raise\' expects an object derived from the BaseException class. Attempting to '
+                     'raise any other object will lead to an error.',
+            'E0711': '\'NotImplemented\' is strictly used as a return value for binary special methods '
+                     '(e.g __eq__), indicating that they\'re not implemented. You mean to raise '
+                     '\'NotImplementedError\', which indicates that the abstract method must be '
+                     'implemented by the derived class.',
+            'E0712': '\'except\' expects an object derived from the BaseException class. Attempting to '
+                     'raise any other object will lead to an error.',
+            'W0702': 'It\'s bad practice to use the \'except\' keyword without passing an expression, '
+                     'since any and all expressions will be caught, which may lead to bugs.',
+            'W0703': 'It\'s bad practice to use \'except: Exception\' since any and all expressions '
+                     'will be caught, which may lead to bugs.',
+            'W0705': '\'except\' blocks are analyzed top to bottom, so if we try to catch the same '
+                     'exception multiple times, only the first \'except\' block for the exception '
+                     'will be reached.',
+            'W0706': 'If the \'except\' block uses \'raise\' as it\'s first or only operator, it '
+                     'will just raise back the exception immediately.',
+            'W0711': '\'except\' can catch multiple exceptions, but they have to be passed in as a '
+                     'tuple. Your exception to catch is a result of a binary operation (%s), '
+                     'which will result in only one of the exceptions being caught by the  \'except\' block.',
+            'W0716': '%s is not a valid operation on exceptions.'
+        }
+    },
+    'pylint.checkers.format': {
+        'FormatChecker': {
+            'W0301': '\';\' in Python is used to put multiple statements on a single line, so this '
+                     'semicolon is unnecessary. In general, there is no good reason to ever use a \';\' '
+                     'in Python.',
+            'W0312': 'This code is indented with both %ss and %ss. You should stick to just one, '
+                     'but note that spaces are preferred.',
+            'C0301': 'This line is %s characters long, exceeding our default limit of %s characters.',
+            'C0303': 'Trailing whitespace is poor formatting, and should be removed.',
+            'C0304': 'It\'s proper practice to have the final line in your file be a newline, '
```

Comment:
By convention, each Python file should end with a blank line. To fix this, go down to the bottom of your file and press Enter/Return to add an empty line.